docs/README.md

```diff
@@ -33,42 +37,736 @@ All output will adhere to the same general JSON structure:
+| fields        | *         | CSV of fields to include in response [Learn More](#fields)  |
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+| status        | Published | CSV of statuses [Learn More](#status)  |
+| lang          | *         | Include translation(s) [Learn More](#language)  |
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Retrieved item<br>`meta`: Depends on requested metadata  |
+| 404 Not Found    | `message`: Collection doesn’t exist, or item doesn't exist within collection    |
+
+#### Examples
+
+* Return the project item with an ID of `1`
+  ```bash
+  curl -g https://api.directus.io/_/items/projects/1
+  ```
+
+
+
+
+### Get Items
+Gets items from within a given collection
+
+```http
+GET /items/[collection-name]/
+```
+
+#### Query Parameters
+| Name          | Default   | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| limit         | 200       | The number of items to request                   |
+| offset        | 0         | How many items to skip before fetching results   |
+| sort          | id        | CSV of fields to sort by [Learn More](#sorting)  |
+| fields        | *         | CSV of fields to include in response [Learn More](#fields)  |
+| filter[field] |           | Filter items using operators [Learn More](#filtering)  |
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+| status        | Published | CSV of statuses [Learn More](#status)  |
+| lang          | *         | Include translation(s) [Learn More](#language)  |
+| q             |           | Search string [Learn More](#search-query)  |
+| id            |           | CSV of primary keys to fetch  |
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Array of items<br>`meta`: Depends on requested metadata  |
+| 404 Not Found    | `message`: Collection doesn’t exist    |
 
 #### Examples
 
 * Search for all projects in the `design` category
   ```bash
-  curl -g https://demo-api.getdirectus.com/_/items/projects?filter[category][eq]=design
+  curl -g https://api.directus.io/_/items/projects?filter[category][eq]=design
+  ```
+
+
+
+
+### Update Item
+Update a single item from within a given collection
+
+```http
+PATCH /items/[collection-name]/[primary-key-value]
+PUT /items/[collection-name]/[primary-key-value]
+```
+
+::: warning PATCH VS PUT
+* **PATCH** partially updates the item with the provided data, any missing data is ignored
+* **PUT** fully updates the item and any missing data will fallback to its default value
```

Comment:
~fully updates~ => overwrites

I think the term overwrites expresses the actual functionality better